test/functional/apps/home/_sample_data.js

```diff
@@ -87,15 +106,40 @@ export default function ({ getService, getPageObjects }) {
         const tsvb = await find.existsByCssSelector('.vega-view-container');
         expect(tsvb).to.be(true);
       });
+
+      it.skip('should launch sample logs data set dashboard', async ()=> {
+        await PageObjects.home.launchSampleDataSet('logs');
+        await PageObjects.header.waitUntilLoadingHasFinished();
+        const today = new Date();
+        const todayYearMonthDay = today.toISOString().substring(0, 10);
+        const fromTime = `${todayYearMonthDay} 00:00:00.000`;
+        const toTime = `${todayYearMonthDay} 23:59:59.999`;
+        await PageObjects.header.setAbsoluteRange(fromTime, toTime);
+        const panelCount = await PageObjects.dashboard.getPanelCount();
+        expect(panelCount).to.be(11);
+      });
+
     });
 
     // needs to be in describe block so it is run after 'dashboard describe block'
     describe('uninstall', () => {
-      it('should uninstall sample data set', async ()=> {
+      it('should uninstall flights sample data set', async ()=> {
         await PageObjects.home.removeSampleDataSet('flights');
         const isInstalled = await PageObjects.home.isSampleDataSetInstalled('flights');
         expect(isInstalled).to.be(false);
       });
+
+      it('should uninstall logs sample data set', async ()=> {
+        await PageObjects.home.removeSampleDataSet('logs');
+        await retry.try(async () => {
+          const successToastExists = await PageObjects.home.doesSampleDataSetSuccessfulUninstallToastExist();
```

Comment:
Remove the check for the toast notifications. Those are flaky.

Reply:
++ removed, testing locally (because I just figured out how to do that 😄 )